spire/templates/apps/dovetail-router.yml

```diff
@@ -739,6 +741,25 @@ Resources:
             Effect: Allow
             Principal:
               Service: ecs-tasks.amazonaws.com
+          - Action:
+              - dynamodb:BatchGetItem
+              - dynamodb:ConditionCheck
+              - dynamodb:DescribeTable
+              - dynamodb:DescribeTimeToLive
+              - dynamodb:GetItem
+              - dynamodb:Query
+            Effect: Allow
+            Resource: !Split
```

Comment:
This split/sub/join/split insanity was only needed for a short time when the arrangements table was being moved from a regional table to a global table, and we were accessing both. I don't believe that's the case any more.

So for this I would either simplify this back to a single `!Ref`, or probably `FrequencyDynamodbTableName` should to be renamed to something that indicates that it can/does contain multiple table names